python/lsst/validate/drp/photerrmodel.py

```diff
@@ -169,7 +171,7 @@ def fitPhotErrModel(mag, mag_err):
     matchRef : `int` or `astropy.unit.Quantity, optional
         Should match at least matchRef stars.
 
-    Attributes
+    Attributes of returned `Blob`
```

Comment:
This isn't a permitted section (numpydoc is a stickler). I'd just have a `Returns` section and describe the blob there. Something like:

```rst
Returns
-------
blob : `lsst.verify.Blob`
    Blob with datums:

    - ``brightSnr``: Threshold in SNR for bright sources used in this  model.
    - ``sigmaSys``: Systematic error floor.
    - [..]
```